modules/vba_documents/app/serializers/vba_documents/v2/upload_serializer.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module VBADocuments
+  module V2
+    class UploadSerializer < VBADocuments::UploadSerializer
+      delegate :status, to: :object
+
+      def attributes(fields)
+        attrs = super
+        attrs.delete(:location)
+        attrs
```

Comment:
I haven't tested this, but couldn't this just be:
```suggestion
        super.delete(:location)
```

Reply:
Sadly it doesn't work.  I tried that initially too.  It took me a while to find this magic incantation.